src/controllers/trips.controller.js

```diff
@@ -37,6 +37,51 @@ class tripsController {
       );
     }
   }
+
+  /**
+   * user will recieve all requests he/she  made and data is retrieved from the database
+   * @param {Object} req The request object
+   * @param {Object} res The response object
+   * @returns {Object} A user object with selected fields
+   */
+  static async returnTripController(req, res) {
+    try {
+      const tripId = faker.random.uuid();
+      const createdTrip = await tripsService.CreateTrip(req, req.body, tripId, 'return trip');
+      const userManager = await tripsService.findUserManager(createdTrip.userId);
```

Comment:
Hey @Jajabenit250, for the sake of separating concerns, tripService should not deal with user management. There's a need for another service that should cater for that